### PR TITLE
Inline image attachments detection fixes

### PR DESCRIPTION
**Pull request by pboguslawski**
## Proposed change

Znuny does not recognize inline attachments when formatted with
whitespaces like test1.png in sample e-mail added in this commit:

```
 <IMG
             SRC = '
     cid:part1.E5AE6B6E.C188FB97@host1.test'
 alt="" class="">
```

Thunderbird recognizes it and displays as inline image.

When forwarding HTML messages Znuny skips attachments that have
Content-ID somewhere in HTML message body, not in src attribute.
This may cause poorly attached inline images (like test2.png in sample
e-mail added in this commit) to be absent in forwarded message.

This mod fixes above issues, removes redundant replace operations and
fixes inconsistent CID searching regexps.

## Type of change
- '1 - 🐞 bug 🐞'            - Bugfix (non-breaking change which fixes an issue)

## Additional information

- Related: https://github.com/OTRS/otrs/pull/1256
- Author-Change-Id: IB#1052680

## Checklist

- [x] The code change is tested and works locally.(❗)
- [x] There is no commented out code in this PR.(❕)
- [x] You improved or added new unit tests.(❕)
- [x] Local ZnunyCodePolicy run passes successfully.(❕)
- [x] Local unit tests pass.(❕)
- [ ] GitHub workflow ZnunyCodePolicy passes.(❗)
- [ ] GitHub workflow unit tests pass.(❗)

### CODE REVIEW

**Comment by rkaldung** [comment]
Hi @pboguslawski,

Please follow the [contributing guidelines](https://github.com/znuny/Znuny/blob/rel-6_0-dev/CONTRIBUTING.md) and use the pull request template. And also, please point out if this is a bug fix or a feature.

**Comment by pboguslawski** [comment]
Done.

**Comment by dennykorsukewitz** [comment]
Hi @pboguslawski 

we have changed our rules regarding copyright.
We therefore ask you to remove your newly added copyrights.

```
Znuny is an open fork of an existing software.
So we have to respect the already given copyright of the original creators.

New files will be licensed using the AGPL Version 3. 
If you contribute code to the Znuny project you will get mentioned in the pull request incl. the commit, 
in CHANGES.md and in AUTHORS.md. 
We will not mention you in the file you provided or changed. 
Your work is highly appreciated and acknowledged but you contribute it to the project 
and your copyright will pass on to the fork itself.
```
https://github.com/znuny/Znuny/blob/rel-6_1-dev/CONTRIBUTING.md#licensing-copyright-and-credits

File: scripts/test/PostMaster.t [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # --
 # Copyright (C) 2001-2021 OTRS AG, https://otrs.com/
 # Copyright (C) 2021 Znuny GmbH, https://znuny.org/
+# Copyright (C) 2021 Informatyka Boguslawski sp. z o.o. sp.k., http://www.ib.pl/
```
Comment (dennykorsukewitz) [inline scripts/test/PostMaster.t]:
```suggestion
```

**Review by dennykorsukewitz — CHANGES_REQUESTED** [review]

**Comment by pboguslawski** [comment]
Done.

**Comment by dennykorsukewitz** [comment]
Hi

This branch has conflicts that must be resolved.

Thank you! :rocket:

**Comment by pboguslawski** [comment]
Replaced by https://github.com/znuny/Znuny/pull/386

**Pull request closed without merging**